models/classes/LtiProvider/LtiProvider.php

```diff
@@ -80,9 +80,9 @@ class LtiProvider implements JsonSerializable
     public function __construct(
         string $id,
         string $label,
-        string $key,
-        string $secret,
-        string $callbackUrl,
+        string $key = null,
+        string $secret = null,
+        string $callbackUrl = null,
```

Comment:
You will also need to modify the `getters`, but are you sure it will not break other places expecting to receive an string? Seems dangerous change.